src/Trinity.C/src/Storage/MTHash/MTHash.lock.cpp

```diff
@@ -194,9 +194,9 @@ namespace Storage
 
         auto                   batch_get_lock_action = [&](std::atomic<uint64_t>* ptr)
         {
-            uint64_t comprand = BUCKETLOCK_EIGHT_NOLOCKS;
+            uint64_t comparand = BUCKETLOCK_EIGHT_NOLOCKS;
```

Comment:
I'm not sure about this. My change is just to make it match another instance elsewhere.

Reply:
looking good.